extern/legacy_remove_me/readme.md

```diff
@@ -0,0 +1,17 @@
+### What is this folder?
+
+This folder contains binaries that we want to remove from dynamo, but at the current time cannot - without causing issues for Dynamo hosts.
+
+These files are actually empty .txt files renamed .dll to avoid including any duplicate types which may be accidentally loaded.
+
+* \bin\Assimp32.dll, 
+* \bin\Assimp64.dll,
+* \bin\AssimpNet.dll
+* \bin\Office.dll
+* \bin\SharpDX.Direct3D11.Effects.dll
+* \bin\SharpDX.Toolkit.Compiler.dll
+* \bin\SharpDX.Toolkit.dll
+* \bin\SharpDX.Toolkit.Graphics.dll
+* \bin\stdole.dll
+* \bin\en-US\DynamoPython.resources.dll
```

Comment:
Ah.. Totally forgot about localization resources, hopefully empty binary works on their pipeline as well

Reply:
yeah, I am going to test this some more, worst case is to add an empty resx to the build... easy, but I want to try keeping everything consistent for now if possible.

Reply:
What is/was DynamoPython? Does the resource no longer exist but the main assembly still exists?

Reply:
yeah, the project remains but is emtpy, I am not sure what it was... we'll get rid of it as soon as we can.

Reply:
Ok. Feel free to leave it if it's there. Or if you would like to delete it and replace it with one of these fake ones, you can also do that. Up to you.